taxi/tests.py

```diff
@@ -1,3 +1,114 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from taxi.models import Driver, Manufacturer, Car
+from taxi.forms import DriverCreationForm
+from django.contrib.auth import get_user_model
+from django.urls import reverse
 
-# Create your tests here.
+
+class DriverTestCase(TestCase):
+    def setUp(self):
+        self.manufacturer1 = Manufacturer.objects.create(name="Ivan")
+        self.manufacturer2 = Manufacturer.objects.create(name="Alex")
+        self.car1 = (Car.objects.create
+                     (model="BMW",
+                      manufacturer=self.manufacturer1)
+                     )
+        self.car2 = Car.objects.create(
+            model="Mercedes",
+            manufacturer=self.manufacturer2
+        )
+        self.driver1 = Driver.objects.create(
+            username="driver1",
+            license_number="tes12345"
+        )
+        self.driver2 = Driver.objects.create(
+            username="driver2",
+            license_number="tes23456"
+        )
+
+    def test_car_list(self):
+        self.car1.drivers.set([self.driver1])
+        self.car2.drivers.set([self.driver2])
+        response = self.client.get(CAR_LIST_URL)
```

Comment:
It seems you are using a constant `CAR_LIST_URL` before defining it. It's defined at the end of the file on line 79. This will raise a `NameError` at runtime. Make sure to define your constants before you use them.